usr/share/rear/finalize/RedHatEnterpriseServer/ppc64le/500_rebuild_initramfs.sh

```diff
@@ -33,6 +33,26 @@ sleep 5
 mount -t proc none $TARGET_FS_ROOT/proc
 mount -t sysfs none $TARGET_FS_ROOT/sys
 
+if is_true $BOOT_OVER_SAN ; then
+
+    # Adding multipath config files must be part of the initramfs in order to
+    # for the "root" disk to be a seen as a multipath device.
+
+    # Add multipath option to dracut (real dracut command will be executed later
+    # in this script).
+    dracut_additional_option="$dracut_additional_option -a multipath"
+
+    # create /etc/multipath.conf on the target if it does not exists on the target.
+    if [ ! -f $TARGET_FS_ROOT/etc/multipath.conf ] ; then
+        LogPrint "/etc/multipath.conf not available in target, creating it..."
+        chroot $TARGET_FS_ROOT /bin/bash -c 'PATH=/sbin:/usr/sbin:/usr/bin:/bin mpathconf --enable --user_friendly_names y --find_multipaths y --with_module y --with_multipathd y --with_chkconfig y'
```

Comment:
Instead of onyl the "it should be ok." assumtion
a LogPrint message when running mpathconf failed
would make the code behave better.
E.g. assume for whatever reason the mpathconf binary
wa nor properly restored from the backup or something
else is wrong in the tartget system why mpathconf cannot
be run successfully there.